qiskit_nature/second_q/problems/electronic_structure_problem.py

```diff
@@ -43,6 +43,35 @@ class ElectronicStructureProblem(BaseProblem):
     The attributes `num_particles` and `num_spin_orbitals` are only available _after_ the
     `second_q_ops()` method has been called! Note, that if you do so, the method will be executed
     again when the problem is being solved.
+
+    In the fermionic case the default filter ensures that the number of particles is being
```

Comment:
The way this comes out leaves the code rather narrow 
![image](https://user-images.githubusercontent.com/40241007/180569542-611c20e9-ce7b-4752-8e99-248f998bb4d5.png)

I am wondering of the indenting of note and code-block is somehow ending up with the narrower areas we see. Clearly the note and code blocks are not indented in the html. Can you see if we can improve things a little?

Reply:
I fixed the overall indenting of the block by suppressing a degree of indentation for the note+code in the doctoring. Thanks